src/PrestaShopBundle/Form/Admin/Sell/Product/OptionsType.php

```diff
@@ -0,0 +1,160 @@
+<?php
+/**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ */
+
+namespace PrestaShopBundle\Form\Admin\Sell\Product;
+
+use PrestaShop\PrestaShop\Core\ConstraintValidator\Constraints\TypedRegex;
+use PrestaShop\PrestaShop\Core\Domain\Product\ProductSettings;
+use PrestaShop\PrestaShop\Core\Form\FormChoiceProviderInterface;
+use PrestaShopBundle\Form\Admin\Type\SwitchType;
+use PrestaShopBundle\Form\Admin\Type\TranslatableType;
+use PrestaShopBundle\Form\Admin\Type\TranslatorAwareType;
+use Symfony\Component\Form\Extension\Core\Type\ChoiceType;
+use Symfony\Component\Form\Extension\Core\Type\TextType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\Translation\TranslatorInterface;
+use Symfony\Component\Validator\Constraints\Length;
+
+/**
+ * This form class is responsible to generate the product options form.
+ */
+class OptionsType extends TranslatorAwareType
+{
+    /**
+     * @var FormChoiceProviderInterface
+     */
+    private $productVisibilityChoiceProvider;
+
+    /**
+     * @param TranslatorInterface $translator
+     * @param array $locales
+     * @param FormChoiceProviderInterface $productVisibilityChoiceProvider
+     */
+    public function __construct(
+        TranslatorInterface $translator,
+        array $locales,
+        FormChoiceProviderInterface $productVisibilityChoiceProvider
+    ) {
+        parent::__construct($translator, $locales);
+        $this->productVisibilityChoiceProvider = $productVisibilityChoiceProvider;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function buildForm(FormBuilderInterface $builder, array $options)
+    {
+        $builder
+            ->add('active', SwitchType::class, [
+                'label' => $this->trans('Online', 'Admin.Catalog.Feature'),
```

Comment:
```suggestion
                'label' => $this->trans('Online', 'Admin.Global'),
```
Hi, this string already exists in Admin.Global, we better keep the existing one. 😊

Reply:
I saw Admin.Catalog.Feature was used in previous product template and Admin.Global :D

Reply:
Hi :) As the string does not already exist in the Admin.Catalog.Feature domain, I think it is better to use Admin.Global, to avoid creating a similar string in another domain and multiply the number of duplicates. Plus, the term is quite general so it is not surprising for it to be located in Admin.Global. @LouiseBonnard do you agree?

Reply:
Absolutely, let's keep it in _Admin.Global_, thanks @Julievrz & @zuk3975!